extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetInstructions.java

```diff
@@ -33,7 +32,17 @@
  */
 public abstract class ParquetInstructions implements ColumnToCodecMappings {
 
-    private static volatile String defaultCompressionCodecName = CompressionCodecName.SNAPPY.toString();
+    public static final String DEFAULT_COMPRESSION_CODEC_NAME = "SNAPPY";
+    public static final int DEFAULT_MAXIMUM_DICTIONARY_KEYS = 1 << 20;
+    public static final int DEFAULT_MAXIMUM_DICTIONARY_SIZE = 1 << 20;
+
+    public static final int MIN_TARGET_PAGE_SIZE = 1 << 11; // 2KB
```

Comment:
See my suggestions. I think we should be using ALL_CAPS_SNAKE_CASE for the actual constants. I would delete the DEFAULT_MIN and DEFAULT_DEFAULT_TARGET declarations and expose the actual constants, since the defaults are useless in a production context. That is, I like my second suggestions more than the first suggestions.